changedetectionio/static/js/browser-steps.js

```diff
@@ -454,7 +454,7 @@ $(document).ready(function () {
         w = window.open(this.href, "_blank", "width=640,height=480");
         const t = $(event.currentTarget).data('type');
 
-        const url = browser_steps_fetch_screenshot_image_url + `&step_n=${step_n}&type=${t}`;
+        const url = window.origin + browser_steps_fetch_screenshot_image_url + `&step_n=${step_n}&type=${t}`;
```

Comment:
hmmm this wont work for people running on the reverse-proxy setup with the install on a sub-path.. the "base url" is only really for notifications i think hmm unsure

Reply:
Oh, right... I forgot about these... Let me think about this more and see what can be done... ;)

Reply:
there's a whole bunch of `const xyz=/full-path/...` defined already in the JS, they are generated with the correct flask `url` function so its always 100% correct according to the environment... if that helps.. you could add something